src/hotspot/share/jvmci/vmStructs_jvmci.cpp

```diff
@@ -221,6 +221,7 @@
   nonstatic_field(JavaThread,                  _vthread,                                      OopHandle)                             \
   nonstatic_field(JavaThread,                  _scopedValueCache,                             OopHandle)                             \
   nonstatic_field(JavaThread,                  _anchor,                                       JavaFrameAnchor)                       \
+  nonstatic_field(JavaThread,                  _lock_id,                                      int64_t)                               \
```

Comment:
These exported fields are used in the inlined monitorenter/exit JIT code, e.g., 
https://github.com/oracle/graal/pull/9366/files#diff-25609d0f9d08c0d8a8e13f1f0b11c996f9fbe3f632a4a099bd1ee634a6748eb3R578-R579
ported from
https://github.com/openjdk/loom/pull/210/files#diff-318d0e76b9a97e8cf8936be1de34e52735c4d947a77cac38babbbf9a081d16fcR540-R542